studio/tartifact_store.py

```diff
@@ -116,7 +116,8 @@ def put_artifact(
 
             tarp = subprocess.Popen(['/bin/bash', '-c', tarcmd],
                                     stdout=subprocess.PIPE,
-                                    stderr=subprocess.STDOUT)
+                                    stderr=subprocess.STDOUT,
+                                    close_fds=True)
```

Comment:
why do we need close_fds here?

Reply:
Yes, if we dont, we run into too many open file descriptor errors eventually